apps/emqx/src/emqx_config.erl

```diff
@@ -788,3 +781,138 @@ to_atom_conf_path(Path, OnFail) ->
                     V
             end
     end.
+
+-spec maybe_init_default_zone() -> skip | ok.
+maybe_init_default_zone() ->
+    case emqx_config:get([zones], ?CONFIG_NOT_FOUND_MAGIC) of
+        ?CONFIG_NOT_FOUND_MAGIC ->
+            skip;
+        Zones when is_map(Zones) ->
```

Comment:
Why not pass the `Zones` to `init_default_zone` so we don't need to call the `get([zones])` twice?

Reply:
thanks, I will merge these two funs into one.